domains/anmail.is-local.org.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "Personal website and email forwarding ",
+
+  "domain": "is-local.org",
+  "subdomain": "anmail",
+
+  "owner": {
+    "repo": "https://github.com/ririko5834/register",
+    "email": "wolfo89+anmail@protonmail.com"
+  },
+
+  "record": {
+    "NS": ["ns1.he.net", "ns2.he.net", "ns3.he.net", "ns4.he.net", "ns5.he.net"]
```

Comment:
Please provide a reason as to why you need NS records.

Reply:
Want to manage DNS records at one place of all my domains/subdomains from different providers. Arent NS records available?